From 08c53df33fba378116ae4cf61e107407537ed761 Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Fri, 14 Nov 2014 13:45:22 -0500
Subject: [PATCH] Move helper tools to usr/lib

This is more in line with the Fedora filesystem standards.
---
 cloudinit/config/cc_keys_to_console.py | 2 +-
 setup.py                               | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/cloudinit/config/cc_keys_to_console.py b/cloudinit/config/cc_keys_to_console.py
index ed7af69..e8bc33a 100644
--- a/cloudinit/config/cc_keys_to_console.py
+++ b/cloudinit/config/cc_keys_to_console.py
@@ -26,7 +26,7 @@ from cloudinit import util
 frequency = PER_INSTANCE
 
 # This is a tool that cloud init provides
-HELPER_TOOL = '/usr/lib/cloud-init/write-ssh-key-fingerprints'
+HELPER_TOOL = '/usr/libexec/cloud-init/write-ssh-key-fingerprints'
 
 
 def handle(name, cfg, _cloud, log, _args):
diff --git a/setup.py b/setup.py
index bd41bc9..dbc9105 100755
--- a/setup.py
+++ b/setup.py
@@ -149,7 +149,7 @@ setuptools.setup(name='cloud-init',
       data_files=[(ETC + '/cloud', glob('config/*.cfg')),
                   (ETC + '/cloud/cloud.cfg.d', glob('config/cloud.cfg.d/*')),
                   (ETC + '/cloud/templates', glob('templates/*')),
-                  (USR + '/lib/cloud-init',
+                  (USR + '/libexec/cloud-init',
                     ['tools/uncloud-init',
                      'tools/write-ssh-key-fingerprints']),
                   (USR + '/share/doc/cloud-init',
-- 
1.8.3.1

